src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -1,13 +1,8 @@
 ---
-- name: Remove official nginx repository
-  become: yes
-  apt_repository: repo='ppa:nginx/stable' state=absent
-  when: ansible_distribution_version == '14.04'
-
 - name: Add nginx repository
   become: yes
   apt_repository: repo='ppa:ondrej/nginx' state=present
-  with_items: "{{ 'ppa:ondrej/nginx' if ansible_distribution_version == '14.04' else 'ppa:nginx/stable' }}"
+  with_items: 'ppa:nginx/stable'
```

Comment:
this is pre-existing, but this doesn't doesn't do anything. There's no evaluation of an item here, so I think `ondrej/nginx` is always being installed

Reply:
Oh wow, thank you @emord for pointing that out. Looks like I misread this task multiple times. The change I was trying to do was

```
apt_repository: repo='ppa:nginx/stable' state=present
```

The current commit doesn't provide anything of value so I'm just going to amend it. Thanks!